PostPairsTrading.md

```diff
@@ -29,14 +22,25 @@ The concept we're referring to is **cointegration** (although there are other me
 > 💡 Which should not be confused with correlation; cointegration is a statistical property of two-time series, indicating a long-term relationship between them despite short-term fluctuations. Cointegrated series move together over time, sharing a common stochastic drift. On the other hand, correlation measures the strength and direction of the linear relationship between two variables at a specific point in time. While correlation captures the degree of association between variables, cointegration reflects a deeper, underlying relationship that persists over time.
 
 Hence, we're interested in **cointegrated assets**, which are assets that exhibit the following characteristics:
+* They have a similar trend, meaning the difference between both assets maintains a constant mean, and this difference fluctuates around that same mean.
+* This inherent relationship persists in the long run, meaning that our series is not dependent on time.
+Before delving into this search for cointegrated pairs, let me introduce you the tick architecture.
+
+## What is the Tick Architecture?

```

Comment:
I think that this section comes a bit out of nowhere. Maybe we should mention that we are going to use the Tick Architecture in the introduction?

Reply:
There is already a paragraph that refers to the tick architecture in the introduction, isn't it?

Reply:
You are right. I guess I missed it